CoreScriptsRoot/Modules/Settings/Utility.lua

```diff
@@ -1412,10 +1409,10 @@ local function ShowAlert(alertMessage, okButtonText, settingsHub, okPressedFunc,
 		Size = UDim2.new(0.95, 0, 0.6, 0),
 		Position = UDim2.new(0.025, 0, 0.05, 0),
 		Font = Enum.Font.SourceSansBold,
-		FontSize = Enum.FontSize.Size36,
+		TextSize = Enum.TextSize.Size36,
```

Comment:
I don't think TextSize is an enum, you should just directly set this member to 36